### DISCUSSION TITLE
Karte für Gesamtzeitraum

### DISCUSSION THREAD

**Discussion by valentin-rosset** [Traewelling/traewelling#3048] [Ideas / Feature Requests]
Mir persönlich fehlt in Träwelling die Funktion, eine Karte anzeigen zu lassen, bei der nicht nur die Fahrten von einem Tag, sondern alle bisher eingecheckten Fahrten angezeigt werden, um eine Gesamtübersicht der befahrenen Strecken zu erhalten

**Comment by jheubuch** [comment]
Duplicate of #2050 ?
